common/src/main/java/com/wynntils/features/chat/GuildRankReplacementFeature.java

```diff
@@ -34,10 +34,12 @@ public class GuildRankReplacementFeature extends Feature {
     public final Config<RankType> rankType = new Config<>(RankType.NAME);
 
     // Test suite: https://regexr.com/7f8kh
+    // Test method at: test/java/TestRegexs$testGuildMessagePattern
```

Comment:
When we move the tests from regexr.com into our test base, we should instead remove these comments completely. The assumption should then always be that the regex is tested in it's proper test location in the test suite.

These comments were only needed since we had external testing.